crates/macros/src/lib.rs

```diff
@@ -90,28 +90,25 @@ pub fn build(stream: proc_macro::TokenStream) -> proc_macro::TokenStream {
             fn copy_to_profile(source: &::std::path::Path, destination: &::std::path::Path, profile: &str) {
                 if let ::std::result::Result::Ok(files) = ::std::fs::read_dir(destination) {
                     for file in files.filter_map(|file| file.ok())  {
-                        if let ::std::result::Result::Ok(file_type) = file.file_type() {
-                            if file_type.is_dir() {
-                                let mut path = file.path();
-                                if let ::std::option::Option::Some(filename) = path.file_name() {
-                                    if filename == profile {
-                                        copy(source, &mut path);
-                                    } else {
-                                        copy_to_profile(source, &path, profile);
-                                    }
+                        let mut path = file.path();
+                        if path.is_dir() {
+                            if let ::std::option::Option::Some(filename) = path.file_name() {
+                                if filename == profile {
+                                    copy(source, &mut path);
+                                } else {
+                                    copy_to_profile(source, &path, profile);
                                 }
                             }
                         }
                     }
                 }
             }
 
-            let mut source : ::std::path::PathBuf = #workspace_dir.into();
+            let mut source: ::std::path::PathBuf = #workspace_dir.into();
             source.push(".windows");
+            println!("cargo:rerun-if-changed={}", source.to_str().expect("`CARGO_MANIFEST_DIR` not a UTF-8 string"));
```

Comment:
I think moving this out the existence check below forces Cargo to rerun every time as it gets confused by the absence of the "changed" directory.

Reply:
Ouch, that's still not fixed?

Reply:
Not as of a month or two when I last checked.